### ISSUE TITLE
Login shell doesn't follow /etc/passwd, and can't change shell.

### ISSUE THREAD

**Issue by DiagonalArg** [fsquillace/junest#226]
My shell is set to bash in /etc/passwd, but when I run "junest" or "junest -f', I get /etc/sh instead.  chsh and usermod do nothing.

**Comment by fsquillace** [comment]
junest defaults to `/usr/bin/sh`. You can explictly call the shell if that helps: `junest -- bash`.

**Comment by DiagonalArg** [comment]
Yes, I see that.  Wouldn't it be sensible for junest to follow /etc/passwd, though?

Take that as a feature request, if you like.

**Comment by fsquillace** [comment]
The shell specified in `/etc/passwd` does not necessarily exist within JuNest.
